scipy/interpolate/interpolate.py

```diff
@@ -2633,293 +2625,6 @@ def fromspline(cls, xk, cvals, order, fill=0.0):
         return cls(sivals, xk, fill=fill)
 
 
-def _dot0(a, b):
-    """Similar to numpy.dot, but sum over last axis of a and 1st axis of b"""
-    if b.ndim <= 2:
-        return dot(a, b)
-    else:
-        axes = list(range(b.ndim))
-        axes.insert(-1, 0)
-        axes.pop(0)
-        return dot(a, b.transpose(axes))
-
-
-def _find_smoothest(xk, yk, order, B=None):
```

Comment:
Might be too soon remove it yet. This one is used by splmake, which is deprecated but sadly used in the wild (via spline).